solidity-v1/dashboard/src/components/modal/threshold/WithdrawGrantedTokens.jsx

```diff
@@ -0,0 +1,197 @@
+          </form>
+          <OnlyIf condition={grants.length > numberOfGrantsDisplayed}>
+            <Button
+              className="withdraw-granted-tokens__view-more-button"
+              onClick={() =>
+                setNumberOfGrantsDisplayed(numberOfGrantsDisplayed + 5)
+              }
+            >
+              <Icons.Add />
+              &nbsp;View More
+            </Button>
+          </OnlyIf>
+        </div>
+      </ModalBody>
+      <ModalFooter>
+        <SubmitButton
+          className="btn btn-primary btn-lg mr-2"
+          type="submit"
+          onSubmitAction={(awaitingPromise) => {
+            onWithdrawClick(awaitingPromise)
+          }}
+          disabled={grants.length > 1 && !formik.values.selectedGrantId}
+        >
+          withdraw
+        </SubmitButton>
+        <Button className={`btn btn-unstyled text-link`} onClick={onClose}>
+          Cancel
+        </Button>
+      </ModalFooter>
+    </>
+  )
+})
+
+const renderGrant = (grant, totalNumberOfGrants, formik) => {
+  return (
+    <div
+      key={`Grant-${grant.id}`}
+      style={{
+        display: "flex",
+        alignItems: "flex-start",
+      }}
+      className={"withdraw-granted-tokens__grants-accordion-container"}
+    >
+      <OnlyIf condition={totalNumberOfGrants > 1}>
+        <input
+          className="radio-without-label"
+          type="radio"
+          name="selectedGrantId"
+          value={grant.id}
+          id={`grant-${grant.id}`}
+          checked={formik.values.selectedGrantId === grant.id}
+          onChange={() => {
+            formik.setFieldValue("selectedGrantId", grant.id)
+          }}
+        />
+        <label htmlFor={`grant-${grant.id}`} />
+      </OnlyIf>
+      <Accordion
+        allowZeroExpanded={totalNumberOfGrants > 1}
+        className={"withdraw-granted-tokens__grants-accordion"}
+      >
+        <AccordionItem
+          {...(totalNumberOfGrants === 1
+            ? { dangerouslySetExpanded: true }
+            : {})}
+        >
+          <AccordionItemHeading>
+            <AccordionItemButton>
+              <div className="withdraw-granted-tokens__token-amount">
+                <TokenAmount
+                  amount={grant.readyToRelease}
+                  amountClassName={"h4 text-mint-100"}
+                  symbolClassName={"h4 text-mint-100"}
+                  token={KEEP}
+                />
+              </div>
+              <OnlyIf condition={totalNumberOfGrants > 1}>
+                <span className="withdraw-granted-tokens__details-text">
+                  Details
+                </span>
+                <span className="withdraw-granted-tokens__expand-button" />
+              </OnlyIf>
+            </AccordionItemButton>
+          </AccordionItemHeading>
+          <AccordionItemPanel>
+            <List>
+              <List.Content>
+                <List.Item className="flex row space-between">
```

Comment:
Should't we add `center` class here? Children look centered because they have the same `fontSize` but I think we should add `center` just in case.

Reply:
Fixed in a100ad855908a6ba6edf8f6360edbb6d1ca22ecf